tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -244,6 +245,7 @@ public IScope ActiveScope
         public static void Configure(TracerSettings settings)
         {
             TracerManager.ReplaceGlobalManager(settings?.Build(), TracerManagerFactory.Instance);
+            Tracer.Instance.TracerManager.Start();
```

Comment:
Is there still (_theoretically_) an issue if customers do the (recommended) setup?

```csharp
var settings = new TracerSettings();
settings.ServiceName = "some-custom-name";
Tracer.Configure(settings);
```

Should we do this instead, to be "safe"?

```csharp
public static void Configure(TracerSettings settings)
{
    TracerManager.ReplaceGlobalManager(settings?.Build(), TracerManagerFactory.Instance);
    _ = Tracer.Instance; // ensures Tracer.Instance is set, if it isn't already
    Tracer.Instance.TracerManager.Start();
}
```

AFAICT, there _shouldn't_ be a case where `Tracer.Instance` is null there, but still...

Reply:
Those two statements are equivalent 😛 

```csharp
Tracer.Instance.TracerManager.Start();
```
==
```csharp
var instance = Tracer.Instance;
instance.TracerManager.Start();
```

Reply:
🤦‍♂️ Well... it's amazing how you end up seeing what you _think_ is there sometimes... I'll see myself out.